Rakefile

```diff
@@ -54,6 +54,23 @@ task :run_html_proofer do
   tester.run
 end
 
+PREREQ_REMOTE_FILES = [
+  'https://raw.githubusercontent.com/travis-infrastructure/terraform-config/master/aws-production-2/generated-language-mapping.json'
+]
+
+desc 'Generates ERb-templated /user documents'
+task :gen_user_erb do
+  require 'find'
+
+  PREREQ_REMOTE_FILES.each { |f| `curl -OsSfL #{f}` }
+
+  files = Find.find(File.join(File.dirname(__FILE__), 'user')) do |f|
+    if File.file?(f) && File.fnmatch('*.erb', f)
+      `erb -r json #{f} | tee #{f.gsub(/\.erb$/,'')}`
+    end
+  end
+end
+
```

Comment:
We'd still need the rake task to pull the info down, but if we pull it down, convert to yaml and drop it in the `_data` dir, the accessing of it is then part of standard jekyll.